modo/api.py

```diff
@@ -370,3 +377,28 @@ def enrich_metadata(self):
                     self.update_element(inst_names[ele.name], ele)
                 else:
                     continue
+
+    def stream_cram(
+        self, cram_name: str, region: str = None, output: str = None
```

Comment:
I think you forgot to change `cram_name` to `cram_path` here?
```suggestion
        self, cram_path: str, region: str = None, output: str = None
```

Reply:
You are correct ... always great to have a second and third set of eyes :)